### ISSUE TITLE
What are the training settings on CIFAR-100?

### ISSUE THREAD

**Issue by ChawDoe** [lukemelas/EfficientNet-PyTorch#121]
I'm training a efficientnet-b0 on CIFAR-100. 
I resize the picture to 224 x 224 using Image.BICUBIC.
And I set the initial lr to 0.1, and multiply it by 0.1 every 3 epoches. 
But I only get acc@85.62%. 
But the result on the paper is 88.1%. 
How can I get the similar result?

**Comment by lukemelas** [comment]
I'm afraid I don't believe they've been released. If you want, you can try leaving an issue on the TensorFlow repo. I'm going to close this for now, but if you find the hyperparameters, do post them!

**Comment by g1910** [comment]
Hi @ChawDoe, can you please share details on how many epochs you trained for? And did you use pretrained imagenet model?

**Comment by siarez** [comment]
I'm in the same boat. I haven't managed to train EfficientNet past 60% accuracy on Cifar100.
I want to train from scratch. I have tried B0 and B5.

My best hyper-parameters so far have have been:
SGD with momentum=0.9, Batch size=256, lr=0.01, L2_weight_decay=1e-5
I haven't managed to get higher than %60 test accuracy!
Is that expected? or are my hyper-parameters out of whack?
Besides the poor test performance, it is also much slower to converge than a standard ResNet50

**Comment by Zi-YuanYang** [comment]
Hi, @siarez !

I'm in the same boat, too! I tried to train efficientnet-b0 from scratch but I can only achieve around 52% accuracy on Cifar-100, and my epoch is 200.

I have no idea why the performance is so poor. Do you have any idea about it?

**Comment by siarez** [comment]
@Zi-YuanYang it's been a while, but if I remember correctly, you need to scale the images up to 224x224 (ImageNet size) to get the full performance. Reasons for this are not entirely clear for me.
